package.json

```diff
@@ -69,6 +69,7 @@
     "axe-webdriverjs": "0.4.0"
   },
   "dependencies": {
+    "@okta/courage": "1.14.0",
```

Comment:
you don't want to put it under dependencies, because people out of Okta can't access `@okta`'s artifacts, so, undo this line, I guess.